openapi/nexus.json

```diff
@@ -6195,6 +6301,194 @@
       }
     },
     "schemas": {
+      "BinRangedouble": {
+        "description": "A type storing a range over `T`.\n\nThis type supports ranges similar to the `RangeTo`, `Range` and `RangeFrom` types in the standard library. Those cover `(..end)`, `(start..end)`, and `(start..)` respectively.",
```

Comment:
It's not specifically partof this PR, but the description here could also use some work. We probably shouldn't reference the stdlib in API docs given it might not be clear _which_ stdlib we're referring to. I don't consider this blocking for the PR.